omarchy-plugin/lib/pomo_link/cli.py

```
@@ -0,0 +1,232 @@
+        if os.path.realpath(dest) == os.path.realpath(exec_path):
+            return dest
+        try:
+            os.unlink(dest)
+        except OSError:
+            return dest
+    try:
+        os.symlink(exec_path, dest)
+    except OSError:
+        try:
+            shutil.copy2(exec_path, dest)
+            os.chmod(dest, 0o755)
```

Ruff (0.16.3)

[error] `os.chmod` setting a permissive mask `0o755` on file or directory

(S103)